vizro-core/tests/unit/vizro/models/test_types.py

```diff
@@ -8,113 +8,111 @@
 from vizro.models.types import CapturedCallable, capture
 
 
-@pytest.fixture
```

Comment:
`test_varargs` and `test_positional_only` are now done in `test_invalid_parameter_kind` with `xfail` removed.